### ISSUE TITLE
Pull doesn't retrieve files

### ISSUE THREAD

**Issue by qriviere-PTC** [PTCInc/thingworx-gitbackup-extension#16]
The situation is pretty simple.
In version 4.1.0 for 9.0, my FileRepository is without entities (only the ".git" folder). On GitHub, the branch has folders and entities. When I do Pull, the FileRepository remains empty. In parallel, when I trigger the Pull from Composer with force = true, I see folders and entities in the FileRepository.
In previous version 2, the Pull in the mashup did retrieve the folders/entities from Git. Does this behaviour changed or is it a true issue?

**Comment by cbadici** [comment]
Hi Quentin, I am not able to reproduce your issue and I am testing this extension in 9.1 When you do a Pull from the Mashup it executes the same service as if you would execute it in the Composer. In my scenario, I had the GitBackup repository empty and I had a repository with some entities in bitbucket. After I have created my GitBackup thing, I did a pull in the Mashup and my repository was populated with the files from the Bitbucket repository.
